src/app/node-data/component.ts

```diff
@@ -175,10 +175,13 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
       .pipe(takeUntil(this._unsubscribe))
       .subscribe(_ => (this._nodeDataService.operatingSystemSpec = this._getOperatingSystemSpec()));
 
-    this._settingsService.adminSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
-      const replicas = this.dialogEditMode ? this._nodeDataService.nodeData.count : settings.defaultNodeCount;
-      this.form.get(Controls.Count).setValue(replicas);
-    });
+    this._settingsService.adminSettings
+      .pipe(take(1))
+      .pipe(takeUntil(this._unsubscribe))
```

Comment:
Last thing would be to remove this line as usually we don't keep both together. There is almost no chance to fire unsubscribe before admin settings cache would be returned.

Reply:
done :)